pkg/daemon/rpm-ostree.go

```diff
@@ -100,6 +115,43 @@ func (r *RpmOstreeClient) loadStatus() (*rpmOstreeState, error) {
 	return &rosState, nil
 }
 
+func parseVer(s string) ([]int, error) {
```

Comment:
Rust has totally spoiled me.  I find it soooo hard to write code like this.  In Rust it's just that simple and elegant: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=3709150890266cfee1f09a35c1f7391f

Reply:
colin we're not rewriting openshift in rust. at least... not yet. ;)